VPC-Section-Formatting (#141)
C2Devel/terraform-provider-croccloud: docs/data-sources/vpc.md

Reviewer: argument attributes -- тут все правильно? вроде мы различаем одного от другого

Reviewer: available

Author: С этим словом всё было норм, хотя был лишний артикль + в одну строку

Provides information about a VPC.

> It can be useful when a module accepts the ID of a VPC as

Reviewer: ```suggestion
The data source can be useful when a module accepts the ID of a VPC as an input variable and needs to, for example, determine the CIDR block of that VPC.
```

an input variable and needs to, for example, determine the CIDR block of that VPC.

## Example Usage

### Specific example

> The following example shows how one might accept the ID of a VPC as a variable

Reviewer: how to

* `cidr_block` - (Optional, String) The CIDR block of the desired VPC.
* `dhcp_options_id` - (Optional, String) The ID of the DHCP options set for the desired VPC.
* `filter` - (Optional, [Block](#filter)) One or more name/value pairs to use as filters.
    * _Valid values:_ See supported names and values in [EC2 API documentation][describe-vpcs]
* `id` - (Optional, String) The ID of the specific VPC to retrieve.
* `state` - (Optional, String) The current state of the desired VPC.
    * _Valid values:_ `available`, `pending`
* `tags` - (Optional, Map of strings) Key-value pairs. Must exactly match pairs on the required resource.

### filter

* `name` - (Required, String) The name of the filter.
    * _Constraints:_ Filter names are case-sensitive
* `values` - (Required, List of strings) One or more filter values.
    * _Constraints:_ Filter values are case-sensitive

## Attribute reference

### Supported attributes

This data source will complete the data by populating any fields that are not included in the configuration with the data for the selected VPC.

In addition to all arguments above, the following attributes are exported:

> * `arn` - (String) The Amazon Resource Name (ARN) of VPC.

Reviewer 2: ```suggestion
* `arn` - (String) The Amazon Resource Name (ARN) of the VPC.
```

VPCs in the current region. The given filters must match exactly one
VPC whose data will be exported as attributes.

* `cidr_block` - (Optional, String) The CIDR block of the desired VPC.

> * `dhcp_options_id` - (Optional, String) The ID of the DHCP options for the desired VPC.

Reviewer: в ресурсе ниже это ... DHCP options set ...

* `filter` - (Optional, List of strings) One or more name/value pairs to use as filters.
    * _Valid values:_ See supported names and values in [EC2 API documentation][describe-vpcs]
* `id` - (Optional, String) The ID of the specific VPC to retrieve.
* `state` - (Optional, String) The current state of the desired VPC.
    * _Valid values:_ `available`, `pending`
* `tags` - (Optional, Map of strings) Key-value pairs. Must exactly match pairs on the required resource.

## Attribute Reference

### Supported attributes

> All attributes except for the `filter` blocks are also exported as

Reviewer: выше где-то осталось "argument attributes"

Author: Поправил

In addition to all arguments above, the following attributes are exported:

* `arn` - (String) The Amazon Resource Name (ARN) of VPC.

> * `enable_dns_support` - (Bool) Whether the VPC has DNS support.

Reviewer: ```suggestion
* `enable_dns_support` - (Boolean) Indicates whether the VPC has DNS support.
```

* `main_route_table_id` - (String) The ID of the main route table associated with this VPC.

> The `cidr_block_associations` attribute block is exported with the following attributes:

Reviewer: блоки же оформляем как подразделы. т.е. здесь по идее надо добавить описание и ссылку на раздел, а подпункты вынести в раздел
